public/sass/components/_footer.scss

```diff
@@ -59,3 +60,15 @@
     }
   }
 }
+
+@include media-breakpoint-down(sm) {
```

Comment:
we already have a @include media-breakpoint-up(sm) {  for this rule, so would be bettter to change the default style  and update existing media query (think mobile first, so default styles are for the mobile screens, and then media queries for larger screens )

Reply:
You're right. I just want to make fewer changes to the code. I changed it.